script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -26,22 +27,27 @@ The earth itself erupted. You'd be forgiven for thinking a bomb had gone off.
 [sha:31a2d2135a66b7f81bd20088656aea4a539bd2cf]{
 -- Page 660, Offset 10426.
 -- 木っ端微塵の力技。ヴローヴの冷気が吹雪なら、アルクェイドの所行は大地の津波だ。
-The power of an overwhelming number of fragments. If Vlov's cold air is like a blizzard, then what Arcueid did is like an earth tsunami.
+The power of an overwhelming number of fragments.
+ If Vlov's cold air is like a blizzard,
+ then what Arcueid did is like an earth tsunami.
 }
 [sha:afef4e00d09328d323a22a176d37427de0f746d6]{
 -- Page 661, Offset 10427.
 -- 飛翔する氷の槍など、この壁の如き土砂の前では飛沫にすぎない。
-A flying ice spear is nothing more than a flung droplet of water before this wall-like agglomeration of earth and sand.
+A flying ice spear is like flicking droplet of water before this wall of earth and sand.
```

Comment:
flicking `a` droplet, but you'd also need to alter the 'before this wall' a bit I think. If we're gonna tone this one down then maybe just
> A flying ice spear is nothing more than a droplet of water before this...